docs/azure-monitor-logging.md

```diff
@@ -0,0 +1,81 @@
+# Azure Monitor tagging
+
+All telemetry sent to Azure Monitor, whether traces, metrics, or spans, should follow this guide.
+
+## Overloading of default dimensions
+
+A few built-in Azure Monitor fields are overloaded:
+
+| field | overload |
+| - | - |
+| `cloud_RoleName` | Either `device` or `service` |
```

Comment:
why a mix of snake and camel case for the field name?

Reply:
These are defined by app insights.  I did it primarily to promote device/service to a top level attribute before I learned the "project" keyword.  Based on doc, this is the appropriate usage for these fields.